python/mxnet/ndarray/sparse.py

```diff
@@ -247,6 +247,9 @@ class CSRNDArray(BaseSparseNDArray):
     row i are stored in indices[indptr[i]:indptr[i+1]] and their corresponding values are stored
     in values[indptr[i]:indptr[i+1]].
 
+    The indices are expected to be sorted per row in ascending order. Duplicate matrix entries are
```

Comment:
nit: "The column indices for a given row are expected to be in ascending order..."